pkg/k8spm/k8spm.go

```diff
@@ -80,13 +86,17 @@ func (m *K8sWorkloadMgr) CreateAppInst(ctx context.Context, clusterInst *edgepro
 	}
 
 	updateSender.SendStatus(edgeproto.UpdateTask, "Deploying App Instance")
-	switch deployment := app.Deployment; deployment {
+	// TODO: We should handle helm the same as a
+	// manifest-based appinst inside the workload manager,
+	// so that the surrounding handlers for namespaces and
+	// policies are common to both.
```

Comment:
Can you open an issue for this. I think this is a good idea, but we'll forget unless we track this.

Reply:
Yep, I had opened https://github.com/edgexr/edge-cloud-platform/issues/420